doc/common/glossary.md

```diff
@@ -20,144 +20,193 @@ To help guide the reader, this glossary provides an introduction to the terminol
-- **Cloud Infrastructure Profile**: The combination of the Cloud Infrastructure Software Profile and the Cloud Infrastructure Hardware Profile that defines the capabilities of the Cloud Infrastructure.
-- **Cloud Infrastructure Software Profile**: defines the behaviour, capabilities and metrics provided by a cloud infrastructure Software Layer
-- **Cloud Infrastructure Software Configuration**: a set of settings (Key:Value) that are applied/mapped to **cloud infrastructure** SW deployment.
+
+- **Hypervisor:** a software that abstracts and isolates workloads with their own operating systems from the underlying physical resources. Also known as a virtual machine monitor (VMM).
+
+- **Instance:** is a virtual compute resource, in a known state such as running or suspended, that can be used like a physical server.
+    >_*Note:*_ Can be used to specify VM Instance or Container Instance.
+
+- **Network Function (NF):**  functional block or application that has well-defined external interfaces and well-defined functional behaviour.
+  - Within **NFV**, a **Network Function** is implemented in a form of **Virtualised NF** (VNF) or a **Cloud Native NF** (CNF).
+
+
+- **Network Function Virtualisation (NFV):** The concept of separating network functions from the hardware they run on by using a virtual hardware abstraction layer.
+
+- **Network Function Virtualisation Infrastructure (NFVI):** The totality of all hardware and software components used to build the environment in which a set of virtual applications (VAs) are deployed; also referred to as cloud infrastructure.
+ >_*Note:*_ The NFVI can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. **NFVI** and **VNF** are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
+
+- **Network Service (NS):** composition of **Network Function**(s) and/or **Network Service**(s), defined by its functional and behavioural specification, including the service lifecycle.
+
+- **Software Defined Storage (SDS):** An architecture which consists of the storage software that is independent from the underlying storage hardware. The storage access software provides data request interfaces (APIs) and the SDS controller software provides storage access services and networking.
+
+- **Virtual Application (VA):** A general term for software which can be loaded into a Virtual Machine.
+  >_*Note:*_ a **VNF** is one type of VA.
+
+- **Virtual CPU (vCPU):** Represents a portion of the host's computing resources allocated to a workload. One or more vCPUs can be assigned to a workload.
```

Comment:
technically the vCPUs are assigned to a VM or container (or a virtualised resource).
```suggestion
- **Virtual CPU (vCPU):** Represents a portion of the host's computing resources allocated to a virtualised resource (virtual machine or container). One or more vCPUs can be assigned to a  virtualised resource.
```

Reply:
@EditKoselak and @pgoyal01: i agree with the last version.  @EditKoselak: can you please accept the update or respond?